types/rebass/package.json

```diff
@@ -0,0 +1,6 @@
+{
+    "private": true,
+    "dependencies": {
+        "@emotion/core": "*"
```

Comment:
Should we use `*` for version restriction? If supported version of emotion is clear, I think specifying it would be better.

Reply:
Not sure what's the best practice for requiring these dependencies in DefinitelyTyped. Just saw it like this in another repo - but setting a specific version should also be fine.

Reply:
OK, I was just curious that there might be some version specified clearly in rebass document.